### DISCUSSION TITLE
Is "root" account necessary as Docker user?

### DISCUSSION THREAD

**Discussion by saikek** [huan/docker-simple-mail-forwarder#49] [General]
Is it necessary to run image as root ? 
It brings security risks - if container is compromised, attacker get access to root account on your system.

docs.docker.com/develop/develop-images/dockerfile_best-practices -> user section

It would be better if new user is created just for this service.

**Comment by huan** [comment]
Did you tried to run it without root?

I believe it should be possible, and please let me know is there any problem with it, thanks.

**Comment by fjwbarnard** [comment]
I am getting `Permission denied` error when trying to launch the container:  
```
$ sudo docker run zixia/simple-mail-forwarder test
/bin/bash: /entrypoint.sh: Permission denied
```

NOTE: I do NOT have a _root_ account configured on my ubuntu server
